frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -987,8 +988,27 @@ export const ClusterServiceVersionDetails: React.FC<ClusterServiceVersionDetails
                   isInline
                   className="co-alert"
                   variant="danger"
-                  title={`${status.phase}: ${status.message}`}
-                />
+                  title={
+                    status.reason === CSVConditionReason.CSVReasonCopied
+                      ? t('olm~Operator failed')
+                      : `${status.phase}: ${status.message}`
```

Comment:
I would always show "Operator failed" here and then show `status.message` in the alert body below when reason isn't copied.

Reply:
Thanks, @spadgett.  Updated.